packages/compare/compare.py

```diff
@@ -0,0 +1,89 @@
+# Local imports
+import json
```

Comment:
@echarles is there a way to not repeat this file on the repo src and on this subpackage?

I tried adding an extra include on the root package but they need to be under the same root folder.

I could not bundle this file and copy it on CI maybe?

Just to avoid having two files falling out of sync

Reply:
I only see one `compare.py` in this PR. What do I miss here?

Reply:
The ts file on /packages/compare/src/com..ts. Is a 1-1 cooy of yhe same file in /src/